bedrock/firefox/templates/firefox/election/index.html

```diff
@@ -219,15 +219,15 @@ <h2 class="mzp-c-call-out-title">Get Firefox</h2>
           </div>
           <div id="case-outofdate" class="hidden mzp-c-call-out-cta">
             <div class="mzp-c-call-out-cta-container">
-              <div class="mzp-c-button-download">
+              <div class="c-button-download">
                 <p>Your version of Firefox isn’t compatible with these extensions. Once you’ve updated, please return to this page for the extensions.</p>
-                {{ download_firefox(alt_copy='Update Firefox', dom_id='btn-download-oldfx', button_color='mzp-c-button mzp-t-download') }}
+                {{ download_firefox(alt_copy='Update Firefox', dom_id='btn-download-oldfx') }}
               </div>
             </div>
           </div>
           <div id="case-mobile" class="hidden mzp-c-call-out-cta">
             <div class="mzp-c-call-out-cta-container">
-              <div class="mzp-c-button-download">
+              <div class="c-button-download">
```

Comment:
Note for reviewer: these weren't really download button molecules, and contain a mixture of custom content. Rather than rename everything, I just removed the `mzp` prefix.